.github/workflows/release-connect-npm-init.yml

```diff
@@ -36,15 +61,21 @@ jobs:
         run: yarn install
 
       - name: Build dependencies
-        # Running `build:libs` requires action to checkout with `submodules: true`.
         run: yarn build:libs
 
       - name: Run @trezor/connect create npm release branch
         run: |
-          git config --global user.name "trezor-ci"
-          git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
-          gh config set prompt disabled
-          gh api /user --jq .login
-          node ./ci/scripts/connect-release-init-npm.js ${{ github.event.inputs.semver }}
+          yarn tsx ./ci/scripts/connect-release-npm-init.ts ${{ github.event.inputs.deploymentType }} ${{ env.BRANCH_NAME }}
```

Comment:
This again tries to push the code to the protected branch. Or I'm missing something?

Reply:
No, this is not pushing code to any protected branch. This is triggering the workflow that will initialize the release of NPM.

It requires to have a script that checks what are the dependencies that need to be updated.

Please look at the section `## Release process` in the description of this PR, I am trying to explain there the process. And let' me know if you have doubts or ideas.

Reply:
Ok